src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -38,6 +40,7 @@ const getCaptchaUiSchema = () => {
 const shouldRenderAsRadio = (name) => name.indexOf('methodType') >= 0 || name.indexOf('channel') >= 0;
 
 const createUiSchemaForString = (ionFormField, remediationForm, transformedResp, createUISchema, settings) => {
+  /* eslint-disable no-eval */
```

Comment:
why do we need that?.. Please fix the code instead of ignoring `eslint` until and unless there is no choice

Reply:
fixed. Thanks